README.md

```diff
@@ -124,6 +124,57 @@ Using JSON payload for constructing a message is also available:
     SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
 ```
 
+### Update the message
+
+Using `update-ts: ${{ steps.slack.outputs.ts }}` will update the message that was posted. 
+
+```yaml
+- id: slack
+  uses: slackapi/slack-github-action@v1.18.0
+  with:
+    channel-id: "#channel"
```

Comment:
Although passing the human-readable name works here but we generally recommend using the ID rather than name for long-term stability. Using the name may stop working when someone renames the channel in the future.
```suggestion
    channel-id: 'CHANNEL_ID'
```